student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that you remove the `./` and use  `person` to target the project directory. 
- [ ] Kindly use `require_relative` as the most recommended way. :slightly_smiling_face: 
- Typical Example: :arrow_down: 
`require_relative 'person'`